common/djangoapps/third_party_auth/samlproviderdata/urls.py

```diff
@@ -0,0 +1,11 @@
+"""
+    url mappings for auth/saml/v0/providerdata/
+"""
+
+from rest_framework import routers
+
+from .views import SAMLProviderDataViewSet
+
+samlproviderdata_router = routers.DefaultRouter()
+samlproviderdata_router.register(r'providerdata', SAMLProviderDataViewSet, basename="samlproviderdata")
```

Comment:
nit: prefer snake_case, for the endpoint as well, `provider_data` instead of `providerdata`